kafka/integration_test.go

```diff
@@ -3245,6 +3245,95 @@ func (its *IntegrationTestSuite) TestAdminClient_ListOffsets() {
 
 }
 
+// Test DeleteRecords API which deletes all the records before the specified offset
+// in the particular partition of the specified topic.
+func (its *IntegrationTestSuite) TestAdminClient_DeleteRecords() {
+	t := its.T()
+	bootstrapServers := testconf.Brokers
+	assert := its.Assert()
+
+	// Create a new AdminClient.
+	a := createAdminClient(t)
+	defer a.Close()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	// Create a new topic to test the api and produce some messages to that topic
+	topic := fmt.Sprintf("%s-%d", testconf.TopicName, rand.Int())
+
+	topics := []TopicSpecification{TopicSpecification{Topic: topic, NumPartitions: 3, ReplicationFactor: 1}}
+	createTopicResult, createTopicError := a.CreateTopics(ctx, topics)
+	assert.Nil(createTopicError, "Create Topics should not fail.")
+	assert.Equal(createTopicResult[0].Error.Code(), ErrNoError, "Create Topics Error Code should be ErrNoError.")
```

Comment:
Let's avoid this casing as it's something intermediate between plain English and field/method names.
Also let's break this long line.
```suggestion
	assert.Equal(createTopicResult[0].Error.Code(), ErrNoError,
		"Create topics error code should be ErrNoError.")
```